apps/examples/net_local/net_local_client.c

```diff
@@ -0,0 +1,63 @@
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <stddef.h>
+#include <sys/socket.h>
+#include <sys/un.h>
+#include <errno.h>
+#include <string.h>
+#include <unistd.h>
+
+static const char *server_path = "/dev/server.socket";
+
+extern int client_talk(int sockfd, const char *func, const char *str);
+
+int net_local_client_main(int argc, char *argv[])
+{
+	struct sockaddr_un server_sun;
+	int sockfd;
+	int len;
+
+	printf("[%s] enter\n", __FUNCTION__);
+
+	if ((sockfd = socket(AF_UNIX, SOCK_STREAM, 0)) < 0) {
+		printf("client socket error");
+		return -1;
+	}
+
+	memset(&server_sun, 0, sizeof(server_sun));
+	server_sun.sun_family = AF_UNIX;
+	strcpy(server_sun.sun_path, server_path);
+	len = offsetof(struct sockaddr_un, sun_path) + strlen(server_sun.sun_path);
```

Comment:
```
size_t len = strlen(server_sun.sun_path);
strncpy(server_sun.sun_path, server_path, len);
```
should we use offsetof here?
lwip using whole of structure itself, aligning with lwip will be better.

Reply:
I can't understand this comment. Would you explain more?